### ISSUE TITLE
Warning: Official place is not here

### ISSUE THREAD

**Issue by Neustradamus** [double-commander/doublecmd#7]
Hello @TheNicker, @vitaliyg2, and all,

The official place is here:
- https://github.com/doublecmd/doublecmd

Can you archive and plan to remove your repository and work directly into the official place?

Current forks of this repository: @antikruk, @dandv, @dkmkk, @phongth7

Thanks in advance.

Regards,

Neustradamus

**Comment by dandv** [comment]
Yeah, my fork only exists because of [this PR](https://github.com/double-commander/doublecmd/pull/6) to link to the official repo. I've deleted the fork.

Now that the official repo has migrated the code, I would probably be best to delete or at least archive this repo. 

Thank you @Neustradamus for helping organize things.

**Comment by Neustradamus** [comment]
@dandv: Thanks for your reply, I have not seen your PR before ^^
Yes, it is really better now with the official place and changes done by @alexx2000!
The development will increase, I think :)

**Comment by TheNicker** [comment]
Hello @Neustradamus, 
Thanks for the update about the official DVCS migration and thank you for your efforts for creating a single point of focus.
Having this reality was my intentions for many years, it wasn't possible then, perhaps now it is.

I'm more than happy to work with the "official" repo and get reviews for my PR's, nevertheless, for the near future I intend to keep this repo also active till it'll be evident that the official repo constitutes a stable and a healthy environment.

**Comment by TheNicker** [comment]
> @TheNicker, @vitaliyg2, @dandv, @rich2014, @Skif-off, @alexx2000, @exidler, @wesinator, @iglezz, @decart, @vmaksimov: The problem is always here!
> 
> * https://github.com/double-commander/doublecmd
> * https://github.com/doublecmd/doublecmd

I started this repository long before the official dev team has moved to Github or a modern versioning source control in general.
Nowadays when the official Github repo is alive and active. it has much precedence in search engines over this repo. I don't think people get confused like in the early days.
So what's exactly is the problem ?

Please be advised that this repo is currently being used and it is needed, that's why it is still open, otherwise I would have get rid of it long ago.
Ideally, I would use only the official repo and I wouldn't be writing any source code for double commander, but it is not the case. 
For me double commander is a work tool, in recent times I'm working with Linux as well, so I'm relying on it to provide me what I need. 

I would be happy to discuss about needed features and get feedback, it is okay to have different opinions and needs, and if someone don't like my idea it's fine as long as I am **not depended on one's mindset**, but till then I would require a solution now.

Ultimately it's people, not everyone sees everything, and it's a challenge to transfer ideas to a great precision to one another, not the least in the medium named Github

**Comment by dandv** [comment]
I've unstarred the "bad" repo (honestly, 2 years later, I don't remember the whole story, but the "bad" repo only has 83 stars). However, the "bad" repo has over 10k commits, and they're very recent, so users my get confused.

Who can update the description of the [bad](https://github.com/double-commander/doublecmd) repo on GitHub to indicate its exact nature? Now it's this:

![image](https://github.com/double-commander/doublecmd/assets/33569/654a2ad6-6737-4b2b-b15a-ab97a265ed4b)

**Comment by vmaksimov** [comment]
Guys, 
you have been mistaken with @vmaksimov account, I suppose it has to be @avmaksimov

Best regards, 
Viacheslav Maksimov

**Comment by Neustradamus** [comment]
Dear current @double-commander, and @doublecmd team,

I wait your news.

If you have a problem, I can do all steps to be perfect.

And I will move all tickets into the same repository and redirect from the bad repo to the good repo.

Can you add me like I have requested you at beginning?

Thanks in advance.

**Comment by dandv** [comment]
@Neustradamus: I've deleted my fork. Thanks for helping organize the project!

**Comment by j2969719** [comment]
>I have started this repository long before the official dev team has moved to Github or a modern source versioning control system in general...

im sorry but it doesnt explain why this particular github organization name was chosen (iirc when the official organization already existed here) and why some people who seem to be loosely related to the project proudly flaunt the dc icon in their profile or why the main developer has to spend its time to just show exactly where to send pull requests

**Comment by Neustradamus** [comment]
@j2969719: It is not the official here!

You mix https://github.com/double-commander vs https://github.com/doublecmd

The official is here: https://github.com/doublecmd.

**Comment by avmaksimov** [comment]
> The goal is to have only one!

It's impossible because I have changes that haven't merged.

**Comment by Neustradamus** [comment]
@avmaksimov: I see that your fork is good, it is from the good one "doublecmd".

**Comment by Neustradamus** [comment]
What is the goal of this repository now?

The official is here:
- https://github.com/doublecmd/doublecmd
